### PR TITLE
Update the language for "sort by options" and "added on" field [EOSF-438]

### PR DESCRIPTION
**Pull request by sheriefvt**
## Ticket

https://openscience.atlassian.net/browse/EOSF-438

## Purpose

The "Date Added" dates are the dates the Preprint was created. The Sorting date, however, is really the Date Modified date. There should be consistency to meet user expectations.

## Changes

- Update the translation file. 
- Use the translation service.
- Rename sort buttons to "Sort by Modified Date (newest to oldest)" and "Sort by Modified Date (oldest to newest)"
- Update the "Added on" field to be "Last edited on"

### CODE REVIEW

File: app/controllers/discover.js [hunk 1]
```diff
@@ -60,7 +60,12 @@ export default Ember.Controller.extend(Analytics, {
     queryBody: {},
     providersPassed: false,
 
-    sortByOptions: ['Relevance', 'Upload date (oldest to newest)', 'Upload date (newest to oldest)'],
+    i18n: Ember.inject.service(),
+
+    sortByOptions: Ember.computed('i18n', function() {
```
Comment (binoculars) [inline app/controllers/discover.js]: You're on the right track here

https://github.com/jamesarosen/ember-i18n/wiki/Doc:-Translating-Text#i18nt

This needs to be `Ember.computed('i18n.locale', function`...

Comment (sheriefvt) [inline app/controllers/discover.js]: thanks

**Comment by binoculars** [comment]
@sheriefvt just need to resolve 1 conflict from recent merges into develop

**Comment by brianjgeiger** [comment]
@sheriefvt This needs some merge conflicts resolved. Could you do that and move it back to Passed CR if the tests all pass?

**Comment by sheriefvt** [comment]
@brianjgeiger yes sure

**Pull request closed without merging**